src/main/java/sampletest/Deadlines.java

```diff
@@ -0,0 +1,63 @@
+package sampletest;
+
+import java.time.LocalDate;
+
+/**
+ * CS2103 Individual Project
+ * @author Wei Cheng
+ * Deadlines is a subclass of task which takes an additional variable time
+ */
+public class Deadlines extends Task{
+    private LocalDate date;
+    /**
+     * Constructor for the Deadlines by taking in
+     * a description and time
+     * @param newDescripton description of deadline
+     * @param newTime time of the deadline
+     */
+    public Deadlines(String newDescripton, String newTime){
+        super(newDescripton);
+        this.date = LocalDate.parse(newTime);
+    }
+    /**
+     * Constructor that is used when we are loading the data from the saved file
+     * @param newDescripton description of deadline
+     * @param newTime time of the deadline
+     * @param status status of the deadline
+     */
+    public Deadlines(String newDescripton, String newTime, String status){
+        super(newDescripton, status);
+        this.date = LocalDate.parse(newTime);
+    }
+    /**
+     * return a String representation
+     * of the date
+     * @return String
+     */
+    public String getTime(){
```

Comment:
Instead of manually parsing the components, you can use the `format` method with the `DateTimeFormatter`'s Pattern. You can refer more to [https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/format/DateTimeFormatter.html](url) regarding to the method required to convert your Date to String in your desired format.